libraries/image/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@teachablemachine/image",
-  "version": "0.7.3",
+  "version": "0.8.3",
```

Comment:
0.8.3 because you already "used" the previous ones?

Reply:
there are breaking changes in this API. with the webcam.